mantidproject/mantid
# Made Error Reporter more user friendly

**Description of work.**

- Error reporter text updated with more concise version. 
- Placeholder text in text entry screen disappears when in focus. 
- Privacy policy link moved to more obvious location. 
- MantidDesktopServices now accessible from python.

**To test:**
1. Ensure that the placeholder text disappears on focus. 
2. Check that the privacy policy link launches a browser to the relevant page. 
3. Sending data still works as expected. 

Test steps in plot and workbench, with both a SegFault (using the algorithm) and an exception that does not crash the program but brings up the error reporter. 

Fixes #26459

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
